src/annotator/highlighter.js

```diff
@@ -150,19 +144,48 @@ function drawHighlightsAbovePdfCanvas(highlightEl) {
  * @typedef {HTMLElement & HighlightProps} HighlightElement
  */
 
+/**
+ * Return the complete text nodes inside `range`.
+ *
+ * If a range starts or ends part-way through a text node, the nodes are split
```

Comment:
"the nodes"...meaning the text node that is only partially covered by the range?

Reply:
This should have said "the node is split". I'll correct it.